quad_control/scripts/utilities/jsonable.py

```diff
@@ -45,10 +45,15 @@ def to_string(cls, inner=dict()):
             arg = args[i+1]
             if arg in cls.inner.keys():
                 val = (inner[arg].__name__, inner[arg].to_string())
+            elif type(defs[i]) is np.ndarray:
+                val = str(list(defs[i]))
             else:
                 val = defs[i]
             arg_dic[arg] = val
-        return json.dumps(arg_dic, indent=4)
+        string = json.dumps(arg_dic, indent=4, separators=(', ', ':\n\t'))
+        string = string.replace('"[','[')
+        string = string.replace(']"',']')
```

Comment:
Strings that begin with `[`or end with `]` were originally arrays, so they should be interpreted as such. Therefore the quotes `"` are removed, which turns the string into the original array. The catch is that any string beginning with `[` or ending with `]` will be done the same thing. But there should not be any reasonable risk that somebody uses such strings as parameter values.